ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml

```diff
@@ -57,9 +57,10 @@ images:
       - registry.access.redhat.com/ubi8/ubi:latest
   to: e2e-test
 promotion:
+  additional_images:
+    cincinnati-image: deploy
```

Comment:
`quay.io/openshift/origin-deploy:latest` is a pretty generic name for a Rust-toolset image.  If we are going to push these images to locations that lose the context of "within a single Cincinnati CI run", we should give them names that do a better job standing alone.

Reply:
`deploy` is imagestreamtag name, it would be pushed as `registry.ci.openshift.org/origin/4.11:cincinnati-image`